src/Tribe/Admin/Notice/Plugin_Download.php

```diff
@@ -27,12 +27,16 @@ public function __construct( $plugin_path ) {
 	 * @param string $name         Name of the required plugin
 	 * @param null   $thickbox_url Download or purchase URL for plugin from within /wp-admin/ thickbox
 	 * @param bool   $is_active    Indicates if the plugin is installed and active or not
+	 * @param string $version      Optional version number of the required plugin
+	 * @param bool   $addon        Indicates if the plugin is an add-on for The Events Calendar or Event Tickets
 	 */
-	public function add_required_plugin( $name, $thickbox_url = null, $is_active = null ) {
+	public function add_required_plugin( $name, $thickbox_url = null, $is_active = null, $version = null, $addon = false ) {
```

Comment:
We still need to make sure we add a `@since` that explains these changes.

Reply:
^^